### ISSUE TITLE
Tenants back on the street / cloned again.

### ISSUE THREAD

**Issue by Ludvig11** [loiste-interactive/Obenseuer-Issues#1348]
[Tenants-Outdoors_Lumi_Pakkanen_2.zip](https://github.com/loiste-interactive/Obenseuer-Issues/files/13532991/Tenants-Outdoors_Lumi_Pakkanen_2.zip)
![20231202021954_1](https://github.com/loiste-interactive/Obenseuer-Issues/assets/31344357/c38e4b53-ffe6-499f-8c80-1fa97e51a740)
![20231202022427_1](https://github.com/loiste-interactive/Obenseuer-Issues/assets/31344357/31cc8856-5d83-4573-bbb8-aa502f31e8fd)

**Comment by Oskutin** [comment]
Playtesting version?

**Comment by Ludvig11** [comment]
Yes current [playtesting] but it fixed itself for now, after sleeping.
